### PR TITLE
ci: onboard event streams DA to catalog

### PR DESCRIPTION
**Pull request by akocbek**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: ibm_catalog.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      "name": "deploy-arch-ibm-event-streams",
+      "label": "Event Streams",
+      "product_kind": "solution",
+      "tags": [
+        "event_streams",
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "reference_architecture",
+        "solution"
```
Comment (ocofaigh) [inline ibm_catalog.json:13]: I recently found out that there is only a pre-defined set of tags that you can use. Please remove `event_streams` and `reference_architecture`. Please add `support_ibm` and `integration`

**Comment by ocofaigh** [comment]
@SirSpidey would you mind reviewing the wording in the JSON please?

**Review by SirSpidey — CHANGES_REQUESTED** [review]
The features need more marketing fluff. Description should flesh-out the title.

**Review by SirSpidey — DISMISSED** [review]
content for catalog details in `ibm_catalog.json` looks good.

**Review by SirSpidey — APPROVED** [review]
re-reviewed after rebase

**Pull request merged**